include/range/v3/view/stride.hpp

```diff
@@ -205,10 +205,20 @@ namespace ranges
                         ranges::advance(it, n),
                         ranges::advance(it, n, std::declval<iterator_t<Rng> &>())))
```

Comment:
This conditional `noexcept`  is invalid now - we should just strike it.

Reply:
Yes. I intend to go through range-v3 and strike most conditional `noexcept` clauses that aren't on move constructors, assignment operators, iterator deref, and `iter_move` / `iter_swap`. All the superfluous `noexcept` just make compiles slow for no added benefit.